src/core/cc_sheet.cpp

```diff
@@ -741,37 +741,37 @@ using namespace Parser;
 void Sheet::sheet_round_trip_test()
 {
     {
-        auto blank_sheet = Sheet::Sheet(0);
+        auto blank_sheet = Sheet(0);
```

Comment:
Without this change, the RHS resolves to the name of a constructor. From Xcode 9.3.1:
```
calchart/src/core/cc_sheet.cpp:744:35: Qualified reference to 'Sheet' is a constructor name rather than a type in this context
```